backends-velox/src/main/scala/org/apache/gluten/config/VeloxConfig.scala

```diff
@@ -594,4 +594,11 @@ object VeloxConfig {
       .stringConf
       .checkValues(Set("loose", "strict"))
       .createWithDefault("loose")
+
+  private val COLUMNAR_VELOX_MEMORY_CHECK_USAGE_LEAK =
+    buildConf("spark.gluten.sql.columnar.backend.velox.checkUsageLeak")
```

Comment:
It seem this config is a static conf, we need use `buildStaticConf` to avoid runtime change